supporting/emissions.py

```diff
@@ -434,6 +437,20 @@ def check_geo_types(self):
             
             return 
     
+    def check_polygon_area(self):
+        # Calculate polygon areas in square meters
+        self.geometry['AREA_M2'] = self.geometry['geometry'].area
```

Comment:
I think we need to convert the CRS before this step happens. I just tested this branch with emissions from a CSV file and they are still in degrees when they reach this step. I also suspect that if emissions were input as a shapefile or feather in units of degrees that they do not get projected until after this step too. 

Can you look through the code pipeline and figure out where the emissions are projected? We want to try to minimize how many projections are happening (this slows down the code a lot). One way to do that is recognizing that the emissions are tabular, so each row only needs to be projected once (versus projected per pollutant). A second thing to be sure of is that we don't want to project too many times; if we add the projection to the ISRM grid here, let's make sure we don't also do it downstream.